pkg/standalone/standalone.go

```diff
@@ -134,7 +137,7 @@ func getDaprDir() (string, error) {
 	return p, nil
 }
 
-func runRedis(wg *sync.WaitGroup, errorChan chan<- error, dir, version string, dockerNetwork string) {
+func runRedis(wg *sync.WaitGroup, errorChan chan<- error, dir, version string, dockerNetwork string, installLocation string) {
```

Comment:
This function doesnt amek use of installLocation, so param can be removed.

Reply:
Unfortunately this is needed to fit the signature of the `initSteps` func.

Reply:
nit: It would be good to leave a comment for this.

Reply:
Added